package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "openzeppelin-zos",
-  "version": "2.0.0-rc.3",
+  "name": "openzeppelin-eth",
+  "version": "2.0.0-rc.1",
```

Comment:
Can we keep the same numbering as in the original repo? I know that we can start over being a new name, but I'd rather make it more of a rename.